build.gradle

```diff
@@ -130,13 +142,28 @@ configure(subprojects.findAll { ['embulk-core', 'embulk-deps-cli', 'embulk-deps-
 
     publishing {
         publications {
-            bintrayMavenRelease(MavenPublication) {
+            maven(MavenPublication) {
+                if (project.version.endsWith("-SNAPSHOT")) {
+                    version = "${project.version}-${revision}"
+                }
                 from components.java
                 artifact testsJar
                 artifact sourcesJar
                 artifact javadocJar
             }
         }
+
+        // publishMavenPublicationToGithubRepository
+        repositories {
+            maven {
+                name = "github"
+                url "https://maven.pkg.github.com/embulk/embulk"
+                credentials {
+                    username = project.hasProperty("github_package_user") ? github_package_user : ""
```

Comment:
shall we have in README something about specifying user and token?

Reply:
I don't think we need. This is basically core-team internal only for GitHub Actions. We won't expect individual developers never upload from their environment.

Reply:
Right, just saving time for new core developers ;) Up to you.